src/ripe/order.py

```diff
@@ -82,6 +82,26 @@ def log_order(self, number):
         contents = self.get(url)
         return contents
 
+    def chat_order(self, number):
+        url = self.base_url + "orders/%d/chat" % number
+        contents = self.get(url)
+        return contents
+
+    def chat_lines_order(self, number):
+        url = self.base_url + "orders/%d/chat/lines" % number

```

Comment:
don't you need to pass options to enable the states, for example?

Reply:
Absolutely, forgot it.